components/render/sdl2backend.cpp

```diff
@@ -87,10 +151,18 @@ namespace Render
 
         SDL_GL_SetAttribute(SDL_GL_CONTEXT_PROFILE_MASK, SDL_GL_CONTEXT_PROFILE_CORE);
         SDL_GL_SetAttribute(SDL_GL_CONTEXT_MAJOR_VERSION, 3);
-        SDL_GL_SetAttribute(SDL_GL_CONTEXT_MINOR_VERSION, 2);
+        SDL_GL_SetAttribute(SDL_GL_CONTEXT_MINOR_VERSION, 3);
         SDL_GL_SetAttribute(SDL_GL_DOUBLEBUFFER, 1);
 
         glContext = SDL_GL_CreateContext(screen);
+
+        // Check opengl version is at least 3.3.
+        const GLubyte* glVersion(glGetString(GL_VERSION));
+        int major = glVersion[0] - '0';
+        int minor = glVersion[2] - '0';
+        if (major < 3 || (major == 3 && minor < 3))
+            std::cerr << "ERROR: Minimum OpenGL version is 3.3. Your current version is " << major << "." << minor << std::endl;
```

Comment:
I woudl abort() here. Maybe this whole print could be a message passed to release_assert.